gp_emulator/gpu/computeTranspose.cu

```diff
@@ -0,0 +1,23 @@
+#include "gpu_predict.h"
+//ld_in is the leading dimension of input matrix
+//ld_out is the leading dimension of the output matrix
+extern "C"{
+real *computeTranspose(const real *matrix, const  int ld_in, const  int ld_out)
```

Comment:
prefer `lead_dim_in` to `ld_in`. Longer variable names should not be a problem if you configure your editor/IDE to do name completion.